datafusion/physical-expr/src/window/aggregate.rs

```diff
@@ -171,8 +156,44 @@ impl WindowExpr for AggregateWindowExpr {
     fn uses_bounded_memory(&self) -> bool {
         // NOTE: Currently, groups queries do not support the bounded memory variant.
         self.aggregate.supports_bounded_execution()
-            && !self.window_frame.start_bound.is_unbounded()
             && !self.window_frame.end_bound.is_unbounded()
             && !matches!(self.window_frame.units, WindowFrameUnits::Groups)
     }
 }
+
+impl AggregateWindowExpr for PlainAggregateWindowExpr {
+    fn get_accumulator(&self) -> Result<Box<dyn Accumulator>> {
+        self.aggregate.create_accumulator()
+    }
+
+    /// For a given range, calculate accumulation result inside the range on
+    /// `value_slice` and update accumulator state.
```

Comment:
I suggest documenting the assumption that `cur_range` is always grater than `last_range` (aka the ranges never shrink).